src/Handlers/NetworkTestHandler.ts

```diff
@@ -0,0 +1,47 @@
+import { bind, Handler } from '../Bus';
+import { NetworkTestIntegrationEvent } from './Events';
+import { NetworkTestResult } from '../Integrations';
+import { injectable } from 'tsyringe';
+import { spawn } from 'nexpect';
```

Comment:
Why you don't use the built-in module to spawn processes?

Reply:
this module use it inside but looks like to child process interaction